data/autoyast_sle15/autoyast_sle_powervm_pre.xml.ep

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's remove this `_pre` from the name, in theory is a auto-installation than can be used for anything.